spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinderPropertiesAccessor.java

```diff
@@ -29,7 +29,7 @@
  *
  * @author Gary Russell
  */
-public abstract class AbstractBinderPropertiesAccessor implements BinderProperties {
+public abstract class AbstractBinderPropertiesAccessor {
```

Comment:
I believe this should be renamed `AbstractBindingPropertiesAccessor` now, but it's a bit of a mix between *binder* an *binding* properties. Perhaps we should be splitting those, and maybe even splitting producer vs. consumer binding property accessors? WDYT @garyrussell ?